src/domain/entities/atlassian-design.ts

```diff
@@ -17,17 +17,6 @@ export interface AtlassianProviderUser {
 	readonly id: string;
 }
 
-export class AtlassianAssociation {
```

Comment:
**note:** The whole concept of `AtlassianAssociation` is removed from the domain of this app 🥳 

**CC:** @stannnous